.github/workflows/release-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish Packages
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  publish-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v2
+  
+      - name: Publish to pub.dev
+        uses: bluefireteam/melos-action@v3
+        with:
+          publish: true
+
+      # - name: Create GitHub Release
```

Comment:
What is the plan with this block? Do we want GitHub releases as well or only tags?